packages/io/package.json

```diff
@@ -1,11 +1,10 @@
 {
-  "name": "@jscad/io",
+  "name": "@jscad/io-base",
```

Comment:
we should not be changing package names unless there is a good reason to do so :)

Reply:
I understand. But both packages/io and packages/io/io have the same name. It’s very confusing, so I changed the name here, as this is a ‘private’ module.

Thoughts?

Reply:
err no , packages/io is a folder
packages/io/io is the root package

Reply:
Good. I’m beginning to understand.
So, why does this file exist? (packages/io/package.json)

Reply:
I double checked and I **think** you are right, that file should not exist ! 
I hope I have not missed anything, but I see no reason to keep it :) Good find !

Reply:
removed